### PR TITLE
chore(rust): rename ockam `project authenticate` clap command to ocka…

### LINKED ISSUE #4940: Rename `ockam project authenticate` clap command to `ockam project enroll`
### Current behavior

```sh
ockam project authenticate
```

### Desired behavior

```sh
ockam project enroll
```
Depends on issue https://github.com/build-trust/ockam/issues/4939 being completed first

Update `authenticate` command which can be found [here](https://github.com/build-trust/ockam/blob/develop/implementations/rust/ockam/ockam_command/src/project/authenticate.rs#L0-L1)

Update examples found [within here](https://github.com/build-trust/ockam/tree/develop/examples/rust/get_started/examples)

Update bats testings found [within here](https://github.com/build-trust/ockam/tree/develop/implementations/rust/ockam/ockam_command/tests/bats)

---

_We love helping new contributors!_
_If you have questions or need help as you work on your first Ockam contribution, please leave a comment on [this discussion](https://github.com/ockam-network/ockam/discussions/1081)._
_If you're looking for other issues to contribute to, checkout [this discussion](https://github.com/build-trust/ockam/discussions/3206) and labels -  https://github.com/build-trust/ockam/labels/good%20first%20issue or https://github.com/build-trust/ockam/labels/help%20wanted_

### PR DESCRIPTION
**Pull request by leafoliage**
## Current behavior

```
ockam project authenticate
```

## Proposed changes

```
ockam project enroll
```

Closes #4940 

## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

### CODE REVIEW

**Comment by leafoliage** [comment]
The link to my PR of CLA: https://github.com/build-trust/ockam-contributors/pull/217

File: implementations/rust/ockam/ockam_command/tests/bats/authority.bats [hunk 1]
```diff
@@ -19,7 +19,7 @@ teardown() {
 
   run "$OCKAM" identity create authority
   run "$OCKAM" identity create enroller
-  # m1 will be pre-authenticated on authority.  m2 will be added directly, m3 will be added through enrollment token
+  # m1 will be pre-enrolld on authority.  m2 will be added directly, m3 will be added through enrollment token
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_command/tests/bats/authority.bats]: s/enrolld/enrolled

**Review by etorreborre — COMMENTED** [review]
Thanks @leafoliage! There is a small typo and a directory to remove `ockam_command/src/project/static/authenticate` and we shoul be good to merge.

File: implementations/rust/ockam/ockam_command/tests/bats/authority.bats (hunk 1 above)
Comment (leafoliage) [reply implementations/rust/ockam/ockam_command/tests/bats/authority.bats]: Oops :sweat_smile: fixed it!

**Review by polvorin — DISMISSED** [review]

**Review by polvorin — DISMISSED** [review]

**Review by polvorin — APPROVED** [review]

**Review by polvorin — APPROVED** [review]

**Review by SanjoDeundiak — APPROVED** [review]

**Review by mrinalwadhwa — APPROVED** [review]

**Comment by mrinalwadhwa** [comment]
@leafoliage thank you and congratulations on an awesome first contribution 🎉🥳

**Comment by leafoliage** [comment]
@mrinalwadhwa Thank you too~

**Pull request merged**